src/Module.Server/ModuleData/item_holsters.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<base>
+	<item_holsters>
+	    <item_holster id="katana_left_hip" equip_action="act_equip_sword" equip_action_left_stance="act_equip_sword_left_stance" unequip_action="act_unequip_sword" unequip_action_left_stance="act_unequip_sword_left_stance" show_holster_when_drawn="false" group_name="hip_left_sword_3" holster_skeleton="holster_skeleton_hip_left" base_set="abdomen_front" holster_position="0.120, 0.070, 0.220" holster_rotation_yaw_pitch_roll="-56.180, 70.790, -120.0"/>
```

Comment:
How is the link made between the katana item and that holster position?

Reply:
Good catch.

Holsters are defined in the crafting_templates.xml for crafted items and in the weapons.xml or shields.xml for single piece items that do not use the crafting system.

I would add the new category in there when I change the items on the item repo.

If you like it better I can add the empty one with this PR though.

Reply:
Yes let's just add the empty file